router-watchdog.js

```diff
@@ -58,11 +58,10 @@ function pingEndpoints() {
       if (failCounter >= CONFIG.numberOfFails) {
         print("Too many fails, resetting...");
         failCounter = 0;
-        Timer.clear(pingTimer);
```

Comment:
This should stay. As the code listens for the notification of the switch turning back to on and then turns the timer back

Reply:
re-added the code as requested.